cmd/status/status_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
2020

Reply:
Was this automatically added?  We  might need to fix the script if so.

Reply:
I'm not sure how it was added. I started the branch a few weeks ago following the contributing doc. It is likely I copied it from somewhere.